File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Data/Domain/Collaboration/CollaborationStatusType.vb
''' <summary> Stati della collaborazione durante il processo di WorkFlow </summary>
Public Enum CollaborationStatusType
    ''' <summary> Bozza </summary>
    BZ = 0
    ''' <summary> Inserimento per visione </summary>
    [IN]
    ''' <summary> Da Protocollare </summary>
    DP
    ''' <summary> Finalizzato al protocollo </summary>
    PT
    ''' <summary> Da leggere </summary>
    DL

    ''' <summary>
    ''' Non Previsto
    ''' </summary>
    ''' <remarks>Casistica di errore: indica che la collaborazione si trova in uno stato sconosciuto alla procedura.</remarks>
    NP

    ''' <summary>
    ''' Non Trovata
    ''' </summary>
    ''' <remarks>Casistica di errore: indica che la collaborazione con l'id specificato non è stata trovata.</remarks>
    NT
    ''' <summary>
    ''' Collaborazione speciale
    ''' </summary>
    ''' <remarks>Collaborazione speciale</remarks>
    WM
End Enum

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Entity/Collaborations/CollaborationStatusType.cs
namespace VecompSoftware.DocSuiteWeb.Entity.Collaborations
{
    public class CollaborationStatusType
    {
        /// <summary>
        /// Bozza
        /// </summary>
        public const string Draft = "BZ";
        /// <summary>
        /// Inserimento per visione/firma
        /// </summary>
        public const string Insert = "IN";
        /// <summary>
        /// Da Protocollare
        /// </summary>
        public const string ToProtocol = "DP";
        /// <summary>
        /// Finalizzato al Protocollo
        /// </summary>
        public const string Registered = "PT";
        /// <summary>
        /// Da leggere
        /// </summary>
        public const string ToRead = "DL";
        /// <summary>
        /// Stato non previsto
        /// </summary>
        /// <remarks>Casistica di errore</remarks>
        public const string NotExpected = "NP";
        /// <summary>
        /// Non trovata
        /// </summary>
        /// <remarks>Casistica di errore</remarks>
        public const string NotFound = "NT";
        /// <summary>
        /// Collaborazione speciale
        /// </summary>
        public const string NoManage = "WM";
    }
}
